plugins/module_utils/azure_rm_common.py

```diff
@@ -1415,30 +1437,26 @@ def _get_env_credentials(self):
 
         return None
 
-    # TODO: use explicit kwargs instead of intermediate dict
-    def _get_credentials(self, params):
+
```

Comment:
There is a line between functions (there are two blank lines)。

Reply:
fixed